experimental/cloudstrapper/playbooks/roles/control/files/self_sign_certs.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# self_sign_certs.sh generates a set of keys and self-signed certificates.
+#
+# Generated secrets
+#   - rootCA.key, rootCA.pem -- certs for trusted root CA (rootCA.key is
+#     subsequently deleted)
+#   - controller.key, controller.crt -- certs for orc8r deployment's public
+#     domain name, signed by rootCA.key
+#
```

Comment:
We can delete this I guess

Reply:
This file is not required anymore. Deleted